test/e2e/jobs/jobs.go

```diff
@@ -184,5 +201,12 @@ func UpgradeAndTestRetinaAdvancedMetrics(kubeConfigFilePath, chartPath, valuesFi
 
 	job.AddScenario(latency.ValidateLatencyMetric())
 
+	job.AddScenario(windows.ValidateWindowsBasicMetric())
```

Comment:
Nit: probably doens't make a difference since windows is only basic metrics, but any reason why this test is being done after the upgrade to advanced metrics and not in basic metrics test above?